automerge/src/lib.rs

```diff
@@ -38,12 +39,14 @@ mod op_set;
 mod op_tree;
 mod query;
 mod sync;
+pub mod transaction;
```

Comment:
Does this module need to be part of our public API or can we just expose the `Transaction` type?